### ISSUE TITLE
Hyperparameter beta settings

### ISSUE THREAD

**Issue by yyang318** [zaixizhang/CBD#2]
Nice work. I am reading the paper and the code. In the overall objective function (8) in the paper, a weighting hyperparameter beta is applied to the last term. May I ask how this beta is used in the code? Thanks.

**Comment by zaixizhang** [comment]
Hi,
Thanks for your interest in our work! Beta is the hyperparameter in the loss function. It can be implemented with a weight decay in optimization.
